x-pack/plugins/apm/server/routes/transactions/get_throughput_charts/index.ts

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
@sqren Do you think it's worth moving this file from `lib` to `routes/transactions/` right now?

Reply:
Probably better to wait for now, and instead create a separate PR that moves all of it at once.